src/control/projectControl.js

```diff
@@ -0,0 +1,34 @@
+/* eslint-disable linebreak-style */
+/* eslint-disable no-undef */
+/* eslint-disable no-shadow */
```

Comment:
This is disabled on the original `.eslintrc.json` that you are supposed to use, so no need to disable it again here: https://github.com/microverseinc/linters-config/blob/master/javascript/.eslintrc.json

Reply:
I've added "linebreak-style": "off" rule since it wasn't in the file. Thanks for point it out.